src/amo/api/abuse.js

```diff
@@ -34,18 +37,24 @@ export type ReportAddonResponse = {|
   |},
   message: string | null,
   reporter: AbuseReporter | null,
+  reporter_name: string | null,
+  reporter_email: string | null,
+  reason: string | null,
 |};
 
 export function reportAddon({
   addonSlug,
```

Comment:
It looks like we aren't sending the slug anymore but either the slug or guid? if that's the case, then maybe we should rename this prop to `addonId`.